templates/article/actualites.html.twig

```diff
@@ -2,6 +2,7 @@
 
 {% block page_title %}{{  current.name }} - Page {{ currentPage }}{% endblock %}
 {% block canonical_url url('articles_list', {category: current.slug, page: currentPage}) %}
+{% block meta_description "Retrouvez l'ensemble des " ~ ('actualites.label.' ~ current.slug)|trans ~ " publié(e)s sur le site de La République En Marche ! Nous publions dans cette partie du site l'ensemble des actualités liées à notre mouvement et aux réformes menées par Emmanuel Macron et le Gouverment pour transformer la France. La page actuelle est la numéro #{currentPage}." %}
```

Comment:
`La page actuelle est la numéro` -> `La page actuelle est le numéro`?

Reply:
`La page actuelle est la numéro X` is the correct syntax. @JulHb has confirmed the sentence :+1:

Reply:
🤐